### PR TITLE
Ading custom password generator

### PR DESCRIPTION
**Pull request by nilasini**
Create custom random password generator.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: CustomPwdGenerator/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# CustomPwdGenerator
+Create custom random password generator
```
Comment (vihanga-liyanage) [inline CustomPwdGenerator/README.md]: Please add a complete readme file following the root readme instructions.

File: CustomPwdGenerator/sample-custom-password-generator/pom.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            <url>http://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+    </repositories>
+
+
```
Comment (vihanga-liyanage) [inline CustomPwdGenerator/sample-custom-password-generator/pom.xml]: Shall we remove the additional new line?

File: CustomPwdGenerator/sample-custom-password-generator/src/main/java/com/wso2/custom/password/generator/internal/CustomPasswordGeneratorDSComponent.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                null);
+        log.info("CustomPasswordGeneratorDSComponent bundle activated successfully..");
+    }
+
+    protected void deactivate(ComponentContext context) {
+        if (log.isDebugEnabled()) {
+            log.debug("CustomPasswordGeneratorDSComponent is deactivated ");
+        }
+    }
+
```
Comment (vihanga-liyanage) [inline CustomPwdGenerator/sample-custom-password-generator/src/main/java/com/wso2/custom/password/generator/internal/CustomPasswordGeneratorDSComponent.java]: Shall we remove the additional new line?

File: CustomPwdGenerator/sample-custom-password-generator/pom.xml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                        </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
```
Comment (vihanga-liyanage) [inline CustomPwdGenerator/sample-custom-password-generator/pom.xml]: Please add a new line to the EOF.

File: CustomPwdGenerator/sample-custom-password-generator/pom.xml [hunk 5]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (madurangasiriwardena) [inline CustomPwdGenerator/sample-custom-password-generator/pom.xml]: add license

**Pull request merged**